A Light CNN for Deep Face Representation with Noisy Labels/README.md

```diff
@@ -0,0 +1,136 @@
+
+# Light CNN for Deep Face Recognition, in PyTorch
+A [PyTorch](http://pytorch.org/) implementation of [A Light CNN for Deep Face Representation with Noisy Labels](https://arxiv.org/abs/1511.02683) from the paper by Xiang Wu, Ran He, Zhenan Sun and Tieniu Tan.  The official and original Caffe code can be found [here](https://github.com/AlfredXiangWu/face_verification_experiment).
+
+[DAGsHub Repository](https://dagshub.com/Bharat-mtr/LightCNN)
+
+### Table of Contents
+- <a href='#updates'>Updates</a>
+- <a href='#installation'>Installation</a>
+- <a href='#datasets'>Datasets</a>
+- <a href='#training'>Training</a>
+- <a href='#evaluation'>Evaluate</a>
+- <a href='#performance'>Performance</a>
+- <a href='#citation'>Citation</a>
+- <a href='#references'>References</a>
+
+## Updates
+
+- Feb 9, 2022
+    - Light CNN v4 pretrained model is released.
+- Jan 17, 2018
+	- Light CNN-29 v2 model and training code are released. The `100% - EER` on LFW achieves **99.43%**.
+	- The performance of set 1 on MegaFace achieves **76.021%** for rank-1 accuracy and **89.740%** for TPR@FAR=10^-6. 
+- Sep 12, 2017
+	- Light CNN-29 model and training code are released. The `100% - EER` on LFW achieves **99.40%**.
+	- The performance of set 1 on MegaFace achieves **72.704%** for rank-1 accuracy and **85.891%** for TPR@FAR=10^-6. 
+- Jul 12, 2017 
+	- Light CNN-9 model and training code are released. The `100% - EER` on LFW obtains **98.70%**.  
+	- The performance of set 1 on MegaFace achieves **65.782%** for rank-1 accuracy and **76.288%** for TPR@FAR=10^-6. 
+- Jul 4, 2017
+	- The repository was built.
+
+## Installation
+- Install [pytorch](http://pytorch.org/) following the website.
+- Clone this repository.
+	- Note: We currently only run it on Python 2.7.	
+
+## Datasets
+- Download face dataset such as  CASIA-WebFace, VGG-Face and MS-Celeb-1M.
+	- The MS-Celeb-1M clean list is uploaded: [DAGsHub Link](https://dagshub.com/Bharat-mtr/LightCNN/src/master/data).
+- All face images are converted to gray-scale images and normalized to **144x144** according to landmarks. 
+- According to the five facial points, we not only rotate two eye points horizontally but also set the distance between the midpoint of eyes and the midpoint of mouth(ec_mc_y), and the y axis of midpoint of eyes(ec_y) .
+- The aligned LFW images are uploaded on [Baidu Yun](https://pan.baidu.com/s/1eR6vHFO).
```

Comment:
This link's download doesn't work for me. Have you succeeded in downloading the actual data? If you have, can you push it to DagsHub

Reply:
pan.baidu has too many restrictions to download files actually.

I don't know how to download the data from here.
Please tell me if there is any other workaround, please let me know.

Thanks

Reply:
Ok, please add a note that this data is hard to access, and that you will appreciate contributions with it, if someone figures it out.

Reply:
I have added the note in readme.

Please merge it if you find it ok.

Thanks